index.bs

```diff
@@ -737,6 +766,51 @@ authorizing an authenticator.
 </div>
 
 
+#### Construct the credential - `constructCredentialCallback` method #### {#constructCredentialCallback}
```

Comment:
I forget the rationale for why this new method is being added.  Can you remind me?

Reply:
because in order to create an object associated with the current global object, we need to be on the "main thread" and so our algs, which have been invoked on a background thread by credman (aka "in parallel"), each return a callback (i've only done #createCredential so far) to credman's alg, who then [queues a task](https://html.spec.whatwg.org/multipage/webappapis.html#queue-a-task) (ie on the main thread) to invoke the callback. see also: https://www.w3.org/2001/tag/doc/promises-guide/#queue-tasks